<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ข้อมูลพนักงาน</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>ข้อมูลพนักงาน</string>
    </property>
    <widget class="QLineEdit" name="txtid">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>รหัสพนักงาน</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ชื่อนามสกุล</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtna">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>เพิ่ม</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ลบ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>เเก้ไข</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>บันทึก</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ยกเลิก</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ออก</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>รหัสพนักงาน</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ชื่อนามสกุล</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
